modules/claims_api/lib/claims_api/ews_vbms_sidekiq.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'claims_api/vbms_uploader'
+
+module ClaimsApi
+  module EwsVBMSSidekiq
+    include SentryLogging
+
+    def upload_to_vbms(evidence_waiver_submission, path)
+      uploader = VBMSUploader.new(
+        filepath: path,
+        file_number: retrieve_veteran_file_number(evidence_waiver_submission: evidence_waiver_submission),
+        doc_type: '295'
```

Comment:
I'm assuming we don't want to upload this as a 295. may be worth asking Kayla to see if she knows the evidence waiver number. I can see if we can locate one in VBMS when we meet today.

Reply:
did we ever check on this?

Reply:
no, I will ask Kayla now